library/Zend/ServiceManager/ServiceManager.php

```diff
@@ -605,12 +605,10 @@ public function create($name)
      */
     private function createDelegatorCallback($delegatorFactory, $rName, $cName, $creationCallback)
     {
-        $serviceManager  = $this;
```

Comment:
I made a rollback change on https://github.com/samsonasik/zf2/commit/68a626a6bc7dd28356340af4d004a2a70ef686ca , and in my local the test is pass now.